src/cljs/rems/administration/create_form.cljs

```diff
@@ -177,13 +177,14 @@
  ::send-form
  (fn [{:keys [db]} [_]]
    (let [edit? (::edit-form? db)
-         form-errors (validate-form-template (get-in db [::form :data]) (:languages db))
+         form-preview-data (build-request (get-in db [::form :data]) (:languages db))
```

Comment:
how about calling this `request` instead of `form-preview-data`?

Reply:
You mean re-naming or?

Reply:
Yeah, renaming the local. Just a small suggestion, not insisting